andela/pippin-ah-backend
# feat(facebookAuth): Handle facebook authentication

#### What does this PR do?
- Handle authentication with facebook

#### Description of Task to be completed?
- [x] Install passport-facebook
- [x] Configure passport to authenticate using facebook
- [x] Save user to database on successfull authentication

#### How should this be manually tested?
- Set FACEBOOK_APP_ID and FACEBOOK_APP_SECRET
- Start the app
- Click [Sign up with Facebook](https://authors-haaven.herokuapp.com/api/v1/users/facebook)
- You should be redirected to facebook
- Login with valid facebook account
- Server will respond with a JWT token and user details

#### What are the relevant pivotal tracker stories?
[#163223759]
